src/components/Amount.tsx

```diff
@@ -50,11 +53,11 @@ export default function Amount({
       {...rest}
     >
       {asset && (
-        <Currency amount={asset.amount} ticker={asset.ticker} icon={true} uuid={asset.uuid} />
+        <Currency amount={asset.amount} isNFTAsset={isNFTAsset} ticker={asset.ticker} icon={true} uuid={asset.uuid} decimals={getDecimals(asset.amount)}/>
       )}
 
       {text && (
-        <Text as="span" isBold={bold}>
+        <Text as="span" isBold={bold} textAlign="right">
```

Comment:
Is it safe for all uses to switch the text alignment?

Reply:
Yup. `text` param is in use in the components in this PR and the swap quote. This `textAlign` change is only noticeable on long text entries because of the wrapping of text overflows. This has the text align to the right instead of left on overflows.